src/patterns/SaleHeaderBanner/SaleHeaderBanner.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import { Meta } from '@storybook/react';
+import SaleHeaderBanner, { SaleHeaderBannerProps } from './SaleHeaderBanner';
+import { AuctionState } from './types';
+import SaleHeaderCountdown from './SaleHeaderCountdown';
+import SaleHeaderBrowseAuctions from './SaleHeaderBrowseAuctions';
+
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
+const meta = {
+  title: 'Patterns/SaleHeaderBanner',
+  component: SaleHeaderBanner,
+} satisfies Meta<typeof SaleHeaderBanner>;
+
+export default meta;
+export const Playground = (props: SaleHeaderBannerProps) => <SaleHeaderBanner {...props} />;
+
+Playground.args = {
+  auctionTitle: 'Modern & Contemporary Art: Online Auction, New York',
+  date: '10:00am EDT, 14 Sep 2024',
+  occuranceLabel: 'Begins',
+  location: 'New York',
+  auctionState: AuctionState.preSale,
```

Comment:
can you set `argTypes` so that the `auctionState` is a dropdown option that's selectable, there should be a few examples using the `TextVariants` that you can copy